keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/Logistic.java

```diff
@@ -9,34 +9,39 @@
 import io.improbable.keanu.tensor.dbl.DoubleTensor;
 import io.improbable.keanu.vertices.dbl.KeanuRandom;
 
+/**
+ * @see "Computer Generation of Statistical Distributions
+ * by Richard Saucier,
+ * ARL-TR-2168 March 2000,
+ * 5.1.14 page 27"
+ */
 public class Logistic implements ContinuousDistribution {
 
-    private final DoubleTensor mu;
-    private final DoubleTensor s;
+    private final DoubleTensor location;
+    private final DoubleTensor scale;
```

Comment:
makes sense since wikipedia and wolfram don't agree (`mu` and `s` vs `m` and `b`)